docs/tutorial.md

```diff
@@ -390,10 +390,10 @@ When Meld starts, edit the right side by e.g. adding something to the first
 line. Then close Meld. You can now inspect the rewritten commit with
 `jj diff -r @-` again and you should see your addition to the first line.
 Unlike `jj squash -i`, which left the content state of the commit unchanged,
-`jj touchup` (typically) results in a different state, which means that
+`jj diffedit` (typically) results in a different state, which means that
 descendant commits may have conflicts.
 
-Other commands for rewriting contents of existing commits are `jj restore -i`,
-`jj split`, `jj unsquash -i`. Now that you've seen how `jj squash -i` and
-`jj touchup` work, you can hopefully figure out how those work (with the help of
-the instructions in the diff).
+Other commands for rewriting contents of existing commits are `jj split` and
```

Comment:
Should `jj move` be mentioned here?

Reply:
Yes, that's one of the more important ones, so that would make sense. Thanks.